test/go-tests/test_uniform.go

```diff
@@ -398,8 +401,9 @@ func Test_UniformRegistration_RegistrationOfKeptnIntegrationMultiplePods(t *test
 func Test_UniformRegistration_RegistrationOfKeptnIntegrationRemoteExecPlane(t *testing.T) {
 	// install echo integration
 	// make sure the echo-service uses the same distributor as Keptn core
-	distributorImage, err := GetImageOfDeploymentContainer("lighthouse-service", "distributor")
+	imageName, err := GetImageOfDeploymentContainer("lighthouse-service", "lighthouse-service")
 	require.Nil(t, err)
+	distributorImage := CreateDistributorImage(imageName, "lighthouse-service")
```

Comment:
I believe here we can just use `strings.Replace()` directly, as it's not immediately clear what the `CreateDistributorImage()` function does